server/orders.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+const db = require('APP/db')
+const Order = db.model('orders')
+
+const { mustBeLoggedIn, forbidden } = require('./auth.filters')
+
+module.exports = require('express').Router()
+  .get('/',
+    // The forbidden middleware will fail *all* requests to list orders.
+    // Remove it if you want to allow anyone to list all orders on the site.
+    //
+    // If you want to only let admins list all the orders, then you'll
+    // have to add a role column to the orders table to support
+    // the concept of admin orders.
+    forbidden('listing orders is not allowed'),
+    (req, res, next) =>
+    Order.findAll()
+    .then(orders => res.json(orders))
+    .catch(next))
+  .post('/',
```

Comment:
This whole block (line 17-20) is a little difficult on the eyes... perhaps some formatting/appropriate indenting is in order?